src/Resources/config/services.yml

```diff
@@ -92,6 +92,17 @@ services:
         class: Contao\CoreBundle\Security\User\ContaoUserProvider
         public: false
 
+    contao.store.session:
```

Comment:
The namespace `contao.listener.` is already used for all the other listeners. And since the class is called `StoreSessionListener`, the key must be `store_session`.